packages/runner-ct/src/app/RunnerCt.tsx

```diff
@@ -78,6 +78,16 @@ const App: React.FC<AppProps> = observer(
       window.dispatchEvent(new Event('resize'))
     }
 
+    React.useEffect(() => {
+      eventManager.on('test:before:run', () => {
+        state.plugins[0]?.beforeTest()
```

Comment:
Since it was really difficult to clean-up after the Vue Devtools, no such `unmount` hook like React Devtools, I decided to add a beforeTest and afterTest hook. 

Before each test, we just mount a refresh copy of the devtools over the old one. This seems to work pretty well!